test/fixtures/bpmn/non-regression/labels.03.default.position.bpmn

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_0x0opj6" targetNamespace="http://example.bpmn.com/schema/bpmn" exporter="bpmn-js (https://demo.bpmn.io)" exporterVersion="7.3.0">
+  <bpmn:process id="Process_1" isExecutable="false">
+    <bpmn:startEvent id="StartEvent_1" name="Start Event">
+      <bpmn:outgoing>Flow_1</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:sequenceFlow id="Flow_1" name="horizontal SeqFlow" sourceRef="StartEvent_1" targetRef="Activity_1" />
+    <bpmn:task id="Activity_1" name="Task">
+      <bpmn:incoming>Flow_1</bpmn:incoming>
+      <bpmn:outgoing>Flow_1w8ldp8</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_1w8ldp8" name="vertical SeqFlow" sourceRef="Activity_1" targetRef="Gateway_1vkgxli" />
+    <bpmn:exclusiveGateway id="Gateway_1vkgxli" name="gateway">
+      <bpmn:incoming>Flow_1w8ldp8</bpmn:incoming>
+    </bpmn:exclusiveGateway>
+  </bpmn:process>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
+    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_1">
+      <bpmndi:BPMNEdge id="Flow_1w8ldp8_di" bpmnElement="Flow_1w8ldp8">
+        <di:waypoint x="380" y="139" />
+        <di:waypoint x="380" y="235" />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="BPMNEdge_Flow_1" bpmnElement="Flow_1">
```

Comment:
A flow with label bounds is missing 😄

Reply:
The value of **STYLE_VERTICAL_ALIGN** is not the same if there is label bounds.
And the label position changes in all cases if we change the value of **STYLE_VERTICAL_ALIGN**.

Reply:
Are you sure this comment is related to this thread? I don't get it